app/bundles/CoreBundle/Model/GlobalSearchModalInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Model;
+
+use Doctrine\ORM\Tools\Pagination\Paginator;
+
+interface GlobalSearchModalInterface
```

Comment:
There is a typo, but perhaps dropping the "Model" is even better. We don't care if it is a model or another service as long as it implements the methods described in the interface.
```suggestion
interface GlobalSearchInterface
```